services/evalsrv/service.go

```diff
@@ -68,7 +68,7 @@ func (e *EvalService) EvalCreate(ctx context.Context, projectID, execID string)
 		}
 	}
 
-	addr, err := e.driver.EndpointCreate(ctx, projectID, evalID, execID, exec.Network.HTTPService.InternalPort)
+	addr, err := e.driver.EndpointCreate(ctx, projectID, evalID, execID, evalID, exec.Network.HTTPService.InternalPort)
```

Comment:
I got a bit confused by `evalID` being passed as the `name` here.

I’m **guessing** it’s because an eval will have a dedicated exec to server the `dataset` and `assert` endpoints, and we need and endpoint to serve those endpoints, so we are using the `evalID` as the endpoint name for it.

Am I on the right track?

Reply:
Yeah, the name (which I've changed to the variable 'subdomain') is the part that comes before the domain that it's hosted on. For example: `eval_abc123.unweave.dev`